form-link/src/main/java/com/ritense/formlink/web/rest/impl/CamundaFormAssociationManagementResource.java

```diff
@@ -49,7 +50,14 @@ public CamundaFormAssociationManagementResource(FormAssociationService formAssoc
     @Override
     @GetMapping(params = {"processDefinitionKey"})
     public ResponseEntity<Collection<? extends FormAssociation>> getAll(@RequestParam String processDefinitionKey) {
-        return ResponseEntity.ok(formAssociationService.getAllFormAssociations(processDefinitionKey));
+        final var formAssociations = formAssociationService.getAllFormAssociations(
+            processDefinitionKey
+        );
+        if (formAssociations != null) {
+            return ResponseEntity.ok(formAssociations);
+        } else {
+            return ResponseEntity.notFound().build();
```

Comment:
why not use the `ResponseEntity.noContent().build()` like other endpoints in this resource

Reply:
The security testing will fail otherwise. It tests the following   
`    assertHttpStatus(GET, "/api/form-association-management?processDefinitionKey=1", NOT_FOUND);`